### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Phaneron97**

### CODE REVIEW

File: documentation/tfgd/template.tex
Comment (Sara-Eftekhar-Azam) [inline documentation/tfgd/template.tex:1]: This latest version does not compile for me and gives error. I think there are probably some problems with the files.

**Review by Sara-Eftekhar-Azam — CHANGES_REQUESTED** [review]
The project does not compile for me and gives error.

File: documentation/tfgd/template.tex
Comment (Phaneron97) [reply documentation/tfgd/template.tex:1]: What error are you getting exactly? How did you test this?

Comment (Sara-Eftekhar-Azam) [reply documentation/tfgd/template.tex:1]: I tried to compile and create the pdf life before and it worked before. I get the following error:
[Latex Error.txt](https://github.com/Phaneron97/BDA_proj_rooftiles/files/15053102/Latex.Error.txt)

Comment (Sara-Eftekhar-Azam) [reply documentation/tfgd/template.tex:1]: If it compiles correctly for you and others, maybe it is a problem with my latex. I try to remove the files and try again to see if the problem is solved.

**Comment by Sara-Eftekhar-Azam** [comment]
It is now working correctly and compiles without problem.

**Pull request merged**